centralized-sampling-tests/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.7"
+services:
+  otel:
+    image: amazon/aws-otel-collector:latest
+    command: --config /config/collector-config.yml
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+    volumes:
+      - .:/config
+      - /tmp/awscreds:/tmp/awscreds
```

Comment:
Passes in the aws credentials into docker containers, so that the docker collector can connect to AWS X-ray backend, 
Also used in the aws-otel-java-instrumentation, https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/.github/collector/docker-compose.yml